torch/_dynamo/variables/builder.py

```diff
@@ -1618,10 +1605,27 @@ def wrap_tensor(self, value: torch.Tensor):
                 "requires some design around FSDP + torch.compile."
             )
 
+        # tx.output has multiple tracers if we're introspecting HigherOrderOperator.
+        # When we've discovered an untracked tensor, then we actually need
+        # to get Dynamo to track the tensor (which is what this function does)
+        # and put it as a graph input on the root tracer. Later on,
+        # if the input is actually used in the body of the HigherOrderOperator,
+        # then the relevant SubgraphTracer will lift it to being an input of
+        # the subgraph.
+        # See NOTE [HigherOrderOperator tracing design] for more details.
+
+        example_value = wrap_to_fake_tensor_and_record(
```

Comment:
We have to move the wrap_to_fake_tensor_and_record logic in wrap_fx_proxy out, so that we could provide an example_value whose symints can later be tracked.